src/Table/Table.jsx

```diff
@@ -125,7 +126,11 @@ export class Table extends Component {
               </Cell>
             </tr>
             ) : pages[displayedPageIndex].map(rowData => (
-            <tr className={cssClass.ROW} key={rowIDFn(rowData)}>
+            <tr
+              className={classnames(cssClass.ROW, {rowClickEnabled: onRowClick})}
+              key={rowIDFn(rowData)}
+              onClick={onRowClick}
```

Comment:
do you need to pass the row id?

`onClick={() => onRowClick(rowIDFn(rowData))}`
or
`onClick={(event) => onRowClick(event, rowIDFn(rowData))}`

The first seems sufficient right? Not clear with the component convention is. @osdiab any preferences / thoughts?